.github/pull_requests_template.md

```diff
@@ -0,0 +1,15 @@
+### Description
+
+**Kindly take a look at [CONTRIBUTING.md](https://github.com/scikit-hep/vector/blob/main/.github/CONTRIBUTING.md).**
+
+_Please describe the purpose of this pull request. Reference and link to any relevant issues or pull requests._
+
+## Parameters Before Requesting Reviewer
```

Comment:
I think it would be okay to request a review even if the tests are failing. Sometimes the failing tests are the part where one needs maintainer's help!
```suggestion
## Checklist
```